### DISCUSSION TITLE
Any hook available to get a value from tailwind css ?

### DISCUSSION THREAD

**Discussion by iangregsondev** [tailwindlabs/tailwindcss#3115] [General]
Hi,

Probably a long shot,, but i was wondering if there any hooks or something similar to be able to get the value of a tailwind css value in code ?

For example, mt-8 , I would like to get the attributes of this in code to be able to use it.

My idea is to use tailwind everywhere i can but I am working with some animation libraries and it would be really beneficial to get the values of a css class attributes i.e margin-top etc 

I could hard code those but it would be nice to be able to get these in code 

Any idea if this is possible?

Thanks

**Comment by royvanv** [comment]
Something like [this](https://tailwindcss.com/docs/configuration#referencing-in-java-script)?
